src/Mvc/Mvc.Core/src/Builder/ControllerActionEndpointConventionBuilder.cs

```diff
@@ -42,7 +42,7 @@ public void Add(Action<EndpointBuilder> convention)
     /// <inheritdoc />
     public void Finally(Action<EndpointBuilder> finalConvention)
     {
-        ArgumentNullException.ThrowIfNull(nameof(finalConvention));
+        ArgumentNullException.ThrowIfNull(finalConvention);
```

Comment:
Good catch

Reply:
Credit to the analyzer. I'm doing what it says.